### PR TITLE
New Version of Forgot Password Module

### PR DESCRIPTION
**Pull request by MarkvanMents**
Based on information in Paper document. See [TW-1995]

### CODE REVIEW

File: content/en/docs/appstore/modules/forgot-password.md [hunk 1]
```diff
@@ -27,12 +33,14 @@ If you already use the Forgot Password module in your Mendix version 8 app, you
...
-* [Email Module with Templates](/appstore/modules/email-with-templates/)
+* [Email Connector](/appstore/connectors/email-connector/)¹
 * [Deep Link](/appstore/modules/deep-link/)
 * [Encryption](/appstore/modules/encryption/)
 * [Mx Model Reflection](/appstore/modules/model-reflection/)
 
-## 2 Installing the Forgot Password Module
+¹ Versions of the Forgot Password module below 4.xx (Mendix version 8) and 5.xx (Mendix version 9), and those for Mendix version 7 (3.xx) have a dependency on the deprecated [Email Module with Templates](/appstore/modules/email-with-templates/) module. We recommend that if you are using Mendix versions 8 and above, you upgrade to a later version using the instructions in [Migrate from Email Module with Templates to Email Connector](#migrate-email), below.
+
+## 2 Installing the Forgot Password Module{#installing}
```
Comment (MarkvanMents) [inline content/en/docs/appstore/modules/forgot-password.md:43]: Are there any other changes we should document here?

File: content/en/docs/appstore/modules/forgot-password.md [hunk 2]
```diff
@@ -105,3 +117,79 @@ To convert from Mendix 8.18.x to Mendix 9.12.5 or above, follow the steps below
...
+#### 5.2.4 Actions After the Migration
+
+1. Import the latest version of the Forgot Password module (v4.xx or above for Mendix version 8, or v5.xx or above for Mendix version 9). This will have the Email Connector module as a dependency.
+1. Delete the Email Module with Templates module.
+1. Delete the Email Connector Migration Utility module.
+1. Remove the following Administrator user roles, which were added above:
+    * Email_Connector.EmailConnectorAdmin
+    * MigrationUtility.EmailConnectorMigrationUtilAdmin
+1. Remove any module roles related to Email Module with Templates.
+1. Check the userlib folder in your app directory and remove the Java jars related to the Email Module with Templates module (for example, javax.mail-1.6.2.jar and activation-1.1.jar).
```
Comment (MarkvanMents) [inline content/en/docs/appstore/modules/forgot-password.md]: Can we be clearer which jars need to be deleted?

**Review by MarkvanMents — COMMENTED** [review]
I've added a couple of questions.

File: content/en/docs/appstore/modules/forgot-password.md [hunk 3]
```diff
@@ -50,9 +58,13 @@ In these instructions, it is assumed that your main module is `MyFirstModule`. I
...
-        * Administrator – `Administration.Administrator, DeepLink.Admin, EmailTemplateAdministrator, Encryption.user, ForgotPassword.Administrator, MxModeIReflection.ModeAdministrator. System.Administrator, MyFirstModule.Administrator`
+
+        * Administrator – `Administration.Administrator, DeepLink.Admin, EmailTemplate.Administrator², Encryption.user, ForgotPassword.Administrator, MxModeIReflection.ModeAdministrator. System.Administrator, MyFirstModule.Administrator`
         * Guest – `DeepLink.User, ForgotPassword.Guest_ResetPassword, ForgotPassword.Guest_SignUp, System.User, MyFirstModule.Guest`
-        * User – `Administration.User. DeepLink.User, EmailTemplate.Administrator, ForgotPassword.Guest_ResetPassword, MxModelReflection.Readonly, MxModelReflection.TokenUser. System.User. MyFirstModule.User`
+        * User – `Administration.User. DeepLink.User, EmailTemplate.Administrator², ForgotPassword.Guest_ResetPassword, MxModelReflection.Readonly, MxModelReflection.TokenUser. System.User. MyFirstModule.User`
+
+            ² EmailTemplate permissions are only needed if using a version which uses the deprecated Email Module with Templates module.
```
Comment (SharishmaPodduturi) [inline content/en/docs/appstore/modules/forgot-password.md]: Can we add one more point here?
EmailTemplate permissions are only needed if using a version which uses the deprecated Email Module with Templates module
or else Assign Email_Connector.EmailConnectorAdmin to Administrator if you use Email connector as a dependency.

File: content/en/docs/appstore/modules/forgot-password.md [hunk 4]
```diff
@@ -105,3 +117,79 @@ To convert from Mendix 8.18.x to Mendix 9.12.5 or above, follow the steps below
...
+
+Follow the [Migration Steps](https://github.com/mendixlabs/EmailConnectorMigrationUtility#migration-steps) in the GitHub repo of the community-supported Email Connector Migration Utility tool to configure the migration utility and migrate the data from the Email Module with Templates module to the Email connector module.
+
+#### 5.2.4 Actions After the Migration
+
+1. Import the latest version of the Forgot Password module (v4.xx or above for Mendix version 8, or v5.xx or above for Mendix version 9). This will have the Email Connector module as a dependency.
+1. Delete the Email Module with Templates module.
+1. Delete the Email Connector Migration Utility module.
+1. Remove the following Administrator user roles, which were added above:
+    * Email_Connector.EmailConnectorAdmin
```
Comment (SharishmaPodduturi) [inline content/en/docs/appstore/modules/forgot-password.md]: We should not remove Email_Connector.EmailConnectorAdmin.
We should remove roles related MigrationUtility and Email module with templates

File: content/en/docs/appstore/modules/forgot-password.md (hunk 2 above)
Comment (SharishmaPodduturi) [reply content/en/docs/appstore/modules/forgot-password.md]: @MarkvanMents 
Whichever the jars have .EmailTemplate.RequiredLib file for jar, those need to be deleted
Ex: activation-1.1.1.jar and activation-1.1.1.jar.EmailTemplate.RequiredLib

File: content/en/docs/appstore/modules/forgot-password.md (hunk 3 above)
Comment (SharishmaPodduturi) [reply content/en/docs/appstore/modules/forgot-password.md]: One more update:
In line no 78: Click **SMTP settings** to configure or validate SMTP settings for the [Email with Templates] or Email Connector based on the versions you use

File: content/en/docs/appstore/modules/forgot-password.md (hunk 1 above)
Comment (SharishmaPodduturi) [reply content/en/docs/appstore/modules/forgot-password.md:43]: Versions of the Forgot Password module 4.xx,5.x.x and below 
Mendix version 8-4.0.0
Mendix version 9-5.0.0
Mendix version 7-3.x.x

File: content/en/docs/appstore/modules/forgot-password.md [hunk 5]
```diff
@@ -105,3 +117,79 @@ To convert from Mendix 8.18.x to Mendix 9.12.5 or above, follow the steps below
 1. This will resolve two of the errors. Open the page of error and again change the (master) layout to `Atlas_TopBar(Atlas_UI_Resources)`.
 
 You have now successfully upgraded the Forgot Password Module to work with Mendix version 9.
+
+## 5 Migrating from Email Module with Templates to Email Connector{#migrate-email}
+
+Versions 4.xx (Mendix version 8) and 5.xx (Mendix version 9) and above of the Forgot Password module use the [Email Connector](/appstore/connectors/email-connector/) module rather than the deprecated [Email Module with Templates](/appstore/modules/email-with-templates/) module. There is a community-supported [Email Connector Migration Utility](https://marketplace.mendix.com/link/component/205008) to help with your upgrade to these versions.
```
Comment (SharishmaPodduturi) [inline content/en/docs/appstore/modules/forgot-password.md]: Versions 4.1.0 (Mendix version 8) and 5.1.0 (Mendix version 9) and above ....

File: content/en/docs/appstore/modules/forgot-password.md [hunk 6]
```diff
@@ -105,3 +117,79 @@ To convert from Mendix 8.18.x to Mendix 9.12.5 or above, follow the steps below
...
+    * Email_Connector.EmailConnectorAdmin
+    * MigrationUtility.EmailConnectorMigrationUtilAdmin
+
+#### 5.2.3 Performing Migration
+
+Follow the [Migration Steps](https://github.com/mendixlabs/EmailConnectorMigrationUtility#migration-steps) in the GitHub repo of the community-supported Email Connector Migration Utility tool to configure the migration utility and migrate the data from the Email Module with Templates module to the Email connector module.
+
+#### 5.2.4 Actions After the Migration
+
+1. Import the latest version of the Forgot Password module (v4.xx or above for Mendix version 8, or v5.xx or above for Mendix version 9). This will have the Email Connector module as a dependency.
```
Comment (SharishmaPodduturi) [inline content/en/docs/appstore/modules/forgot-password.md]: versions:
Mx8-4.1.0
Mx9-5.1.0

File: content/en/docs/appstore/modules/forgot-password.md (hunk 1 above)
Comment (SharishmaPodduturi) [reply content/en/docs/appstore/modules/forgot-password.md:43]: from Mx8-4.1.0,Mx9-5.1.0 email connector is a dependency

Comment (MarkvanMents) [reply content/en/docs/appstore/modules/forgot-password.md:43]: Updated: https://github.com/mendix/docs/pull/5539/commits/049a940f51ea9475a2afc7bb43ec905026546681

File: content/en/docs/appstore/modules/forgot-password.md (hunk 3 above)
Comment (MarkvanMents) [reply content/en/docs/appstore/modules/forgot-password.md]: Updated: https://github.com/mendix/docs/pull/5539/commits/049a940f51ea9475a2afc7bb43ec905026546681

File: content/en/docs/appstore/modules/forgot-password.md (hunk 5 above)
Comment (MarkvanMents) [reply content/en/docs/appstore/modules/forgot-password.md]: Updated: https://github.com/mendix/docs/pull/5539/commits/049a940f51ea9475a2afc7bb43ec905026546681

File: content/en/docs/appstore/modules/forgot-password.md (hunk 6 above)
Comment (MarkvanMents) [reply content/en/docs/appstore/modules/forgot-password.md]: Updated: https://github.com/mendix/docs/pull/5539/commits/049a940f51ea9475a2afc7bb43ec905026546681

File: content/en/docs/appstore/modules/forgot-password.md (hunk 4 above)
Comment (MarkvanMents) [reply content/en/docs/appstore/modules/forgot-password.md]: Updated: https://github.com/mendix/docs/pull/5539/commits/049a940f51ea9475a2afc7bb43ec905026546681

File: content/en/docs/appstore/modules/forgot-password.md (hunk 2 above)
Comment (MarkvanMents) [reply content/en/docs/appstore/modules/forgot-password.md]: Updated: https://github.com/mendix/docs/pull/5539/commits/049a940f51ea9475a2afc7bb43ec905026546681

**Review by MarkvanMents — COMMENTED** [review]
I've updated the document to resolve your comments.
Hope this looks good now.

**Comment by MarkvanMents** [comment]
Approved over Slack

**Pull request merged**